index.js

```diff
@@ -7,7 +7,7 @@ const client = new Client();
 const monitoredChannels = new Map();
 
 // in ms
-const bumpInterval = 1 * 1000;
+const bumpInterval = .5 * 1000;
```

Comment:
Half a second?? Are you sure?

Reply:
yes , because the 2 second seems more like 4 seconds

Reply:
I think that might be because it only checks once per second if it should send revival messages